common/src/main/java/tachyon/master/MasterClient.java

```diff
@@ -225,6 +227,28 @@ public synchronized void connect() throws IOException {
   }
 
   /**
+   * Create transport per the connection options Supported transport options are: NOSASL, SIMPLE,
+   * CUSTOM, KERBEROS
+   * @throws TTransportException
```

Comment:
Javadoc says throwing `TTransportException` while the method throws `IOException`.